test.properties

```diff
@@ -7575,7 +7578,11 @@ test.batch.run.property.query[functional-orcllinux9-tomcat90-mysql57-jdk8]=\
         functional-bundle-ubuntu18-tomcat90-mysql57-jdk8,\
         functional-smoke-bundle-wildfly180-mysql57-jdk8,\
         functional-smoke-bundle-wildfly230-mysql57-jdk8,\
-        functional-smoke-bundle-wildfly261-mysql57-jdk8
+        functional-smoke-bundle-wildfly261-mysql57-jdk8,\
+        playwright-js-bundle-tomcat90-mysql57-jdk8
+
+    test.batch.names[portal-release-playwright]=\
```

Comment:
hi @john-co ! Your changes LGTM based on the results for release-tester and the playwright test suite. Just for SF purposes, could you move test.batch.names[portal-release-playwright]=\ into its own section and sort it with the other portal release suites? Similar to https://github.com/liferay/liferay-portal/blob/master/test.properties#L8309-L8316